apps/studio/prisma/schema.prisma

```diff
@@ -32,13 +32,20 @@ model Resource {
   siteId   Int
   site     Site      @relation(fields: [siteId], references: [id])
   parentId Int?
-  parent   Resource? @relation("parentRelation", fields: [parentId], references: [id])
-  blobId   Int?      @unique
-  blob     Blob?     @relation(fields: [blobId], references: [id])
+  parent   Resource? @relation("ParentRelation", fields: [parentId], references: [id])
 
-  children Resource[] @relation("parentRelation")
+  children Resource[] @relation("ParentRelation")
 
   permission Permission[]
+
+  mainBlob   Blob? @relation("MainBlob", fields: [mainBlobId], references: [id])
+  mainBlobId Int?  @unique
+
+  draftBlob   Blob? @relation("DraftBlob", fields: [draftBlobId], references: [id])
+  draftBlobId Int?  @unique
+
+  isPublished Boolean? @default(false)
```

Comment:
nit: really dislike boolean flag explosion, but CLOSE EYE HERE.

[further reading](https://chughgpt.substack.com/p/dont-use-boolean-arguments-use-enums)

Reply:
i can change to enum with a field called state

Reply:
done